### DISCUSSION TITLE
HTML view is not working properly.

### DISCUSSION THREAD

**Discussion by sheikhsherazali** [postalserver/postal#1808] [Help with using Postal]
Hi 

I have upgraded to postal v2.  Please see the status everything is up to the mark

![image](https://user-images.githubusercontent.com/13349939/149045250-3c948747-da5e-4c4e-ad42-9f2dab015d0b.png)

But when I open the HTML page everything is breaking now UI is working. Here is the screenshot

![image](https://user-images.githubusercontent.com/13349939/149045432-eb638f35-a133-41e1-be4d-703d81c6d76f.png)

Please can you help me out in this connection?

**Comment by willpower232** [comment]
When you're upgrading, you need to rewrite the nginx configuration.

Have a look at https://github.com/postalserver/postal/issues/1668#issuecomment-954657653
